data_managers/data_manager_fetch_genome_dbkeys_all_fasta/data_manager/data_manager_fetch_genome_all_fasta_dbkeys.xml

```diff
@@ -1,86 +1,85 @@
+            </when>
+            <when value="directory">
+                <param type="text" name="fasta_filename" value="" label="Full path to FASTA File on disk" optional="False" />
+                <param type="boolean" name="create_symlink" truevalue="create_symlink" falsevalue="copy_file" label="Create symlink to original data instead of copying" checked="False" />
+            </when>
         </conditional>
         <conditional name="sorting">
-          <param name="sort_selector" type="select" label="Sort by chromosome name">
-            <option value="as_is" selected="True">As is</option>
-            <option value="lexicographical">Lexicographical</option>
-            <option value="gatk">GATK</option>
-            <option value="custom">Custom</option>
-          </param>
-          <when value="as_is">
-          </when>
-          <when value="lexicographical">
-          </when>
-          <when value="gatk">
-          </when>
-          <when value="custom">
-            <repeat name="sequence_identifiers" title="Sequence Identifiers" min="1" default="1">
-                <param type="text" name="identifier" value="" label="Sequence Identifier" optional="False" />
-            </repeat>
-            <conditional name="handle_not_listed">
-                <param name="handle_not_listed_selector" type="select" label="How to handle non-specified Identifiers">
-                    <option value="discard" selected="True">Discard</option>
-                    <option value="keep_append">Keep and Append</option>
-                    <option value="keep_prepend">Keep and Prepend</option>
-                </param>
-                <when value="discard">
-                </when>
-                <when value="keep_append">
-                </when>
-                <when value="keep_prepend">
-                </when>
-            </conditional>
-          </when>
+            <param name="sort_selector" type="select" label="Sort by chromosome name">
+                <option value="as_is" selected="True">As is</option>
+                <option value="lexicographical">Lexicographical</option>
+                <option value="gatk">GATK</option>
+                <option value="custom">Custom</option>
+            </param>
+            <when value="as_is">
+            </when>
+            <when value="lexicographical">
+            </when>
+            <when value="gatk">
+            </when>
+            <when value="custom">
+                <repeat name="sequence_identifiers" title="Sequence Identifiers" min="1" default="1">
+                    <param type="text" name="identifier" value="" label="Sequence Identifier" optional="False" />
+                </repeat>
+                <conditional name="handle_not_listed">
+                    <param name="handle_not_listed_selector" type="select" label="How to handle non-specified Identifiers">
+                        <option value="discard" selected="True">Discard</option>
```

Comment:
Why is this a conditional? I know it not your fault but maybe we can improve things here a little bit.